doc/nrf/working_with_nrf/nrf53/thingy53.rst

```diff
@@ -156,57 +148,44 @@ See :ref:`thingy53_gs_updating_firmware` for details about updating firmware ima
 
-The Thingy:53-compatible samples and applications include the MCUboot bootloader that you can use to update firmware.
+Samples and applications built for Thingy:53 include the MCUboot bootloader that you can use to update firmware out of the box.
 This method uses signed binary files :file:`app_update.bin` and :file:`net_core_app_update.bin` (or :file:`dfu_application.zip`).
 You can program the precompiled firmware image using one of the following ways:
 
 * Use the :doc:`mcuboot:index-ncs` feature and the built-in serial recovery mode of Thingy:53.
   In this scenario Thingy is connected directly to your PC through USB.
 * Update the firmware over-the-air (OTA) using Bluetooth LE and the nRF Programmer mobile application for Android or iOS.
-  To use this method, the application that is currently programmed on Thingy:53 must support it.
+  To use this method, the application that is currently programmed on Thingy:53 must support it, refer to :ref:`thingy53_app_fota_smp` section.
   All precompiled images support OTA using Bluetooth.
 
-See :ref:`thingy53_gs_updating_firmware` for the detailed procedures on how to program a Thingy:53 using `nRF Connect Programmer`_ or the nRF Programmer for Android or iOS.
+See :ref:`thingy53_gs_updating_firmware` for the detailed procedures on how to program a Thingy:53 using `nRF Connect Programmer`_ or the `nRF Programmer`_ for Android or iOS.
 
 .. _thingy53_app_guide:
 
 Thingy:53 application guide
 ***************************
 
 The Nordic Thingy:53 does not have a built-in J-Link debug IC.
-Because of that, the samples and applications that are compatible with the Nordic Thingy:53 by default include the MCUboot bootloader with serial recovery support.
-You can update applications compatible with the Nordic Thingy:53 using the DFU functionality with either `nRF Connect Programmer`_ or the `nRF Programmer`_ mobile application.
-
-Enabling DFU and MCUboot requires consistency in configuration of the samples and applications.
-For a sample or application to be compatible with Thingy:53, the application must comply with the configurations described in the following sections.
-
-.. note::
-   To use your application with the Nordic Thingy:53 preprogrammed bootloader, you must only set the proper configuration for the Partition Manager and MCUboot.
-   Configurations that enable the other features are not mandatory, but you can enable them if needed.
-
-   All applications compatible with the Nordic Thingy:53 in the |NCS| also support the additional features.
-   Refer to the source code of the samples and applications for examples of implementation.
+Because of that, Thingy:53 board by default enables MCUboot bootloader with serial recovery support and predefined static Partition Manager memory map.
+It also gives you an option to manually enable FOTA updates.
+Please refer to the following sections for details of what is configured by default and what you can configure by yourself.
```

Comment:
```suggestion
Because of that, the Thingy:53 board enables MCUboot bootloader with serial recovery support and predefined static Partition Manager memory map by default.
You can also enable FOTA updates manually.
See the following sections for details of what is configured by default and what you can configure by yourself.
```